src/routes/content/HelperText.tsx

```diff
@@ -46,128 +48,130 @@ export default function HelperTextPage() {
-            </td>
-            <td>
-              <GoACheckbox checked name="helper-text" disabled={true} />
-            </td>
-            <td>
-              <GoACheckbox checked name="helper-text" disabled={true} />
-            </td>
-            <td>
-              <GoACheckbox checked={false} name="helper-text" disabled={true} />
-            </td>
-            <td>
-              <GoACheckbox checked={false} name="helper-text" disabled={true} />
-            </td>
-            <td>
-              <GoACheckbox checked name="helper-text" disabled={true} />
-            </td>
-          </tr>
-          <tr>
-            <td>Callout</td>
-            <td>
-              <GoACheckbox checked name="helper-text" disabled={true} />
-            </td>
-            <td>
-              <GoACheckbox checked={false} name="helper-text" disabled={true} />
-            </td>
-            <td>
-              <GoACheckbox checked name="helper-text" disabled={true} />
-            </td>
-            <td>
-              <GoACheckbox checked name="helper-text" disabled={true} />
-            </td>
-            <td>
-              <GoACheckbox checked name="helper-text" disabled={true} />
-            </td>
-            <td>
-              <GoACheckbox checked={false} name="helper-text" disabled={true} />
-            </td>
-          </tr>
-          <tr>
-            <td>Details</td>
-            <td>
-              <GoACheckbox checked={false} name="helper-text" disabled={true} />
-            </td>
-            <td>
-              <GoACheckbox checked name="helper-text" disabled={true} />
-            </td>
-            <td>
-              <GoACheckbox checked={false} name="helper-text" disabled={true} />
-            </td>
-            <td>
-              <GoACheckbox checked name="helper-text" disabled={true} />
-            </td>
-            <td>
-              <GoACheckbox checked={false} name="helper-text" disabled={true} />
-            </td>
-            <td>
-              <GoACheckbox checked name="helper-text" disabled={true} />
-            </td>
-          </tr>
-          <tr>
-            <td>Tooltip</td>
-            <td>
-              <GoACheckbox checked={false} name="helper-text" disabled={true} />
-            </td>
-            <td>
-              <GoACheckbox checked name="helper-text" disabled={true} />
-            </td>
-            <td>
-              <GoACheckbox checked={false} name="helper-text" disabled={true} />
-            </td>
-            <td>
-              <GoACheckbox checked name="helper-text" disabled={true} />
-            </td>
-            <td>
-              <GoACheckbox checked={false} name="helper-text" disabled={true} />
-            </td>
-            <td>
-              <GoACheckbox checked name="helper-text" disabled={true} />
-            </td>
-          </tr>
-        </tbody>
-      </GoATable>
+      <div style={{position: "relative", overflowX: "auto"}}> 
```

Comment:
What is the reason for this? Previously there was an issue with the GoATable with horizontal scrolling, but that was recently fixed, so maybe this is no longer necessary.

Reply:
Good to know! I'll pull this out.